src/Gui/AboutApplication.ui

```diff
@@ -457,50 +470,53 @@ p, li { white-space: pre-wrap; }
 &lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;shoogen&lt;/p&gt;
 &lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;siemsiem&lt;/p&gt;
 &lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Skywalker21&lt;/p&gt;
 &lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Starofrainnight&lt;/p&gt;
 &lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Stefan Tröger (ickby)&lt;/p&gt;
 &lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;stoflap&lt;/p&gt;
 &lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;SweeverDev&lt;/p&gt;
+&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Syres916&lt;/p&gt;
 &lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Thomas Schrader&lt;/p&gt;
 &lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;王凌川&lt;/p&gt;
 &lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Timo Seppola&lt;/p&gt;
+&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Tomas Pavlicek (pavltom)&lt;/p&gt;
 &lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;totyg&lt;/p&gt;
 &lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;triplus&lt;/p&gt;
 &lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;trzyha&lt;/p&gt;
 &lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;ulrich1a&lt;/p&gt;
 &lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;UR_&lt;/p&gt;
-&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Uwe Stöhr&lt;/p&gt;
+&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Uwe Stöhr (donovaly)&lt;/p&gt;
```

Comment:
I don't want this. My GitHub name was a stupid thing many years ago. I just had to give an arbitrary name and now I cannot change it. It is not part of y name nor can I even tell you what my Github name would mean (googling gives me a village name in Slovakia :-) ).

I can imagine that others also have a good reason not to input their GitHub names.

Reply:
I have to admit I was sort of curious about "donovaly" :). 

I am in the opposite position: "chennes" is just used at sites that won't let me have a space in my username. I don't mind it being there, but thank you for including my actual name.

Reply:
Fixed!  I take it GitHub does not allow for changing the username, then (I've never tried)?  Also, please be aware, that GitHub publicly connects your username to your real email address.  I'm not sure that the email address would be even removable, because the info is embedded into the FreeCAD commit history.  On the other hand, this info makes it possible for new devs to connect your FC forum posts to your GitHub contributions, which IMO is a good thing so that new people in the forum discussing code changes would know that you speak for changes made by "donovaly".

Also, my motivation for also adding the GitHub username for developers that have made it public was to make it easier for new devs to see who is actually who in FreeCAD.  For me personally, it took a while to realize the connections between GitHub, forum, and real names; even for the people who have made this information public in GitHub and forum, it is not immediately clear who in GitHub is who in the forum.  Of course, some contributors have not made their information public in GitHub nor forum, and this should obviously be respected, and only use the username/pseudonym even if the real name is known; and some people may intentionally even have different usernames between forum and GitHub.

Reply:
> For me personally, it took a while to realize the connections between GitHub

Ok I see the point. I re-thought and you can keep me in with my github name. It would indeed help. For example it took me a while to realize that kunda1 in the forum is luzpaz on Github.

Reply:
Done!